src/views/self/view_teleport.cpp

```diff
@@ -110,20 +110,40 @@ namespace big
 		const auto& selected_ipl = ipls[g.self.ipls.select];
 		if (components::button("LOAD_IPL"_T.data()))
 		{
+			//remove after loading ipls to avoid conflicts
+			for (auto& ipl_name_remove_when_load : selected_ipl.ipl_names_remove_when_load)
+				g_fiber_pool->queue_job([&] {
+					STREAMING::REMOVE_IPL(ipl_name_remove_when_load);
+					LOG(INFO) << "ipls removed to avoid conflicts " << ipl_name_remove_when_load;
+				});
+
 			//unload all previous ipls
 			for (auto& ipl : ipls)
 				for (auto& ipl_name : ipl.ipl_names)
 				{
 					if (STREAMING::IS_IPL_ACTIVE(ipl_name))
```

Comment:
Any native must be called within a fiber, you can't call it from the presentation layer. This was probably there before you touched the code, but it's still wrong and needs to be fixed. The entire code block should be moved into the job, as you're basically queuing up X number of fibers for each active IPL that needs to be removed. One action -> one fiber.